fix: add server setup example

kylo252 opened the pull request:
Add an example of how to configure the setup per server.

Fixes #191

williamboman commented:
@kylo252 I pushed some changes to reflect the new `server:on_ready()` API, wdyt?

kylo252 commented:
> @kylo252 I pushed some changes to reflect the new `server:on_ready()` API, wdyt?

I saw that `opts` weren't obvious that they can be used with the new API, so I added them there. I also switched the variable-names for readability, but feel free to change it back if you think it looks odd.

williamboman commented on README.md:
```diff
@@ -85,15 +87,19 @@ lsp_installer.on_server_ready(function(server)
+        requested_server:setup(opts)
+    end)
+    if not requested_server:is_installed() then
+        requested_server:install()
```
```suggestion
        -- Queue the server to be installed
        requested_server:install()
```

williamboman commented on doc/nvim-lsp-installer.txt:
```diff
@@ -90,6 +86,24 @@ Then, somewhere in your initialization script (see `:h init.lua`): >
+
+    local lsp_installer_servers = require'nvim-lsp-installer.servers'
+
+    local server_avaiilable, requested_server = lsp_installer_servers.get_server("rust_analyzer")
```
```suggestion
    local server_available, requested_server = lsp_installer_servers.get_server("rust_analyzer")
```

williamboman commented on doc/nvim-lsp-installer.txt:
```diff
@@ -90,6 +86,24 @@ Then, somewhere in your initialization script (see `:h init.lua`): >
+    local lsp_installer_servers = require'nvim-lsp-installer.servers'
+
+    local server_avaiilable, requested_server = lsp_installer_servers.get_server("rust_analyzer")
+    if server_avaiilable then
```
```suggestion
    if server_available then
```

williamboman commented on doc/nvim-lsp-installer.txt:
```diff
@@ -90,6 +86,24 @@ Then, somewhere in your initialization script (see `:h init.lua`): >
+            requested_server:setup(opts)
+        end)
+        if not requested_server:is_installed() then
+            requested_server:install()
```
```suggestion
             -- Queue the server to be installed
            requested_server:install()
```

williamboman commented:
Ty @kylo252 !!
